wandb/internal/update.py

```diff
@@ -21,6 +21,8 @@ def _find_available(current_version):
     # Return if no update is available
     pip_prerelease = False
     parsed_current_version = parse_version(current_version)
+    yanked = False
+    deleted = False
     if parse_version(latest_version) <= parsed_current_version:
```

Comment:
Wait, I didn't see this before. My b. The `release_list` logic below is specifically for checking prerelease versions methinks. We need to do our own `release_list` logic before this if where `release_list` gets mutated.

Reply:
Ah I see now, I'll address that